servicex/__init__.py

```diff
@@ -86,9 +86,17 @@ def create_app(test_config=None,
 
         if app.config['OBJECT_STORE_ENABLED']:
             if not provided_object_store:
+                if 'MINIO_SECURED' in app.config:
+                    if isinstance(app.config['MINIO_SECURED'], bool):
```

Comment:
Please update `app.conf.template` to include a default setting for desktop debugging of the app.

Reply:
Already there